README.md

```diff
@@ -98,6 +98,16 @@ In that case, before you run the test, make sure you have created
     `./bin/e2e-appstudio`
    ```
 
+Some tests could requrie you have a Github App created when you run [rhtap-demo](https://github.com/redhat-appstudio/e2e-tests/blob/main/tests/rhtap-demo/rhtap-demo.go) or [build](https://github.com/redhat-appstudio/e2e-tests/blob/main/tests/build/build.go) tests.The command `make local/cluster/prepare` will automatically help you configure Pac with Github App when exporting `E2E_PAC_GITHUB_APP_ID` and `E2E_PAC_GITHUB_APP_PRIVATE_KEY` as envrionemt variables. Regarding more detail about Pac with Github App, you can refer to [Create a Pipelines-as-Code GitHub App](https://pipelinesascode.com/docs/install/github_apps/).
+
+In this case, before you run the tests, make sure the following are ready on Github App.
+   * Put some dummy url to homepage URL (like https://example.com)
+   * Set the webhook as active and also add some dummy (but valid) url there
+   * Select the option to install to any account (so people will be able to install the github app to their github org's repositories)
+   * Select the proper permissions, refer to [Setup Manually](https://pipelinesascode.com/docs/install/github_apps/#setup-manually)
+
+
+
```

Comment:
I mentioned above that 
> I'd move it to Install AppStudio in E2E mode section above (before [this line](https://github.com/redhat-appstudio/e2e-tests/pull/703/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L60)) since these instructions are needed **before running the cluster bootstrap script**

it doesn't make much sense to have these lines here

Reply:
Done.